Composer/packages/client/src/recoilModel/dispatchers/multilang.ts

```diff
@@ -22,6 +23,17 @@ import {
   botDisplayNameState,
 } from './../atoms/botState';
 
+type FileWithId = { id: string };
+
+const filterFiles = (targetFiles: FileWithId[], currentFiles: FileWithId[]) => {
```

Comment:
I think this one needs a better name, from my understanding it returns only new files, so if the name reflects that it would be easier to understand for other devs:
```getNewlyAddedFiles``` (or something like that)

Also please add a comment to the function explaining what it does.